pom.xml

```diff
@@ -10,7 +10,7 @@
   </parent>
 
   <artifactId>cloudbees-folder</artifactId>
-  <version>5.19-SNAPSHOT</version>
+  <version>6.0.0-SNAPSHOT</version>
```

Comment:
Whoah, that seems a little extreme for this change, which IIUC should be compatible enough.